react/Autosuggest/Autosuggest.js

```diff
@@ -152,7 +152,12 @@ export default class Autosuggest extends Component {
           ref={this.storeInputReference}
           {...allAutosuggestProps}
         />
-        {showMobileBackdrop ? <div className={styles.autosuggestBackdrop} /> : null }
+        <div
+          className={classnames({
+            [styles.autosuggestBackdrop]: true,
+            [styles.mobileBackdrop]: showMobileBackdrop
```

Comment:
Not sure if this is personal preference or not, but I try to see our CSS module naming as a natural extension of BEM, but for components — component, element, modifier. So there is no need to reiterate `autosuggest` as we are inside the bounds of the component, the element is the `backdrop` and you have `mobile` modifier. Here's my suggestion for `Autosuggest.less`:
```css
.backdrop { ... }
.backdrop_isMobile { ... }
```
Thoughts?